metrics-datadog/src/main/java/org/coursera/metrics/datadog/DatadogReporter.java

```diff
@@ -201,12 +201,19 @@ private void reportCounter(String name, Counter counter, long timestamp, List<St
         timestamp, host, tags));
   }
 
-  private void reportGauge(String name, Gauge gauge, long timestamp, List<String> tags)
-      throws IOException {
-    final Number value = toNumber(gauge.getValue());
-    if (value != null) {
-      request.addGauge(new DatadogGauge(metricNameFormatter.format(name), value, timestamp, host,
-          tags));
+  // Gauges are the only metrics which can throw exceptions. With a thrown exception all
```

Comment:
Should be a Javadoc-style comment per https://google.github.io/styleguide/javaguide.html#s7.3.4-javadoc-non-required .